karma.conf.js

```diff
@@ -92,6 +92,16 @@ const SL_LAUNCHERS = {
   // },
 };
 
+// Added Chrome --no-sandbox option because
+// https://github.com/travis-ci/docs-travis-ci-com/blob/
```

Comment:
Shorten link to `https://github.com/travis-ci/docs-travis-ci-com/blob/c1da4af0/user/chrome.md#sandboxing` so you can fit it on one line :)

Also, I think I'm a bit confused... Doesn't Travis run our unit tests via Sauce Labs? This config affects the not-Sauce-Labs case, so how does it end up fixing it?